src/routes/request.js

```diff
@@ -10,12 +10,16 @@ const {
   update,
   oneWay,
   multiCityRequest,
-  getRequests
+  getRequests,
+  returnRequest,
+  search
 } = RequestController;
 
 router.post('/multi-city', [auth, validator(requestSchema)], multiCityRequest);
 router.post('/one-way', [auth, validator(requestSchema)], oneWay);
 router.patch('/:requestId', [auth, validator(requestSchema)], update);
 router.get('/', auth, getRequests);
+router.post('/return', [auth, validator(requestSchema)], returnRequest);
+router.get('/search?', auth, search);
```

Comment:
I don't feel it is okay to leave our route this way `router.get('/search?', auth, search)`, change line `21` : `router.get('/', auth, getRequests);` to  `router.get('/userRequests', auth, getUserRequests);`
then yours will be this way below

```suggestion
router.get('/', auth, getRequests);
```

Please ensure that the route you just changed is working properly

Reply:
Noted